app/main.py

```diff
@@ -4,0 +1,98 @@
+import json
+from typing import Any, Callable
+
+from app.car import Car
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> bool or None:
+
+    def error_handler(error_type: type, err_message: str) -> Callable:
+        def wrapper(func: Callable) -> Callable:
+            def inner(*args, **kwargs) -> Any:
+                try:
+                    return func(*args, **kwargs)
+                except error_type:
+                    print(err_message)
+                    return False
+            return inner
+        return wrapper
+
+    err_messages = {
+        "json": "The specified file could not be opened or found.",
+        "object": "Failed to create class instance. "
+                  "Кey not found or invalid value format."
+    }
+
+    @error_handler(IOError, err_messages["json"])
+    def get_json_data(json_file: str) -> dict or bool:
+        with open(json_file, "r") as file:
+            cfg = json.load(file)
+        return cfg
+
+    @error_handler(KeyError or ValueError, err_messages["object"])
```

Comment:
The use of `KeyError or ValueError` is incorrect here. You should use a tuple `(KeyError, ValueError)` to catch multiple exceptions.